ESLint-Formatter.py

```diff
@@ -143,11 +143,64 @@ def get_node_path():
     return node
 
   @staticmethod
+  def get_config_path():
+    cwd = os.getcwd()
+    folders = cwd.split('/')
+    return PluginUtils.get_eslint_config(cwd, folders)
+
+  @staticmethod
+  def get_eslint_config(root, folders):
+    cfs = ['.eslintrc.js', '.eslintrc.yaml', '.eslintrc.yml', '.eslintrc.json', '.eslintrc']
```

Comment:
I don't believe regular configuration files like `.eslintrc` are considered shareable configs in eslint terminology.

From: http://eslint.org/docs/developer-guide/shareable-configs

```
Shareable configs are simply npm packages that export a configuration object.
```